store/ivy.xml

```diff
@@ -17,6 +17,7 @@
   <dependency org="org.javassist" name="javassist" rev="3.18.2-GA" />
   <dependency org="org.slf4j" name="slf4j-api" rev="1.7.36"/>
   <dependency org="org.slf4j" name="slf4j-simple" rev="1.7.36"/>
+  <dependency org="org.apache.logging.log4j" name="log4j-slf4j-impl" rev="2.17.2" />
```

Comment:
In build.xml have defined version 2.17.1 and here 2.17.2

Reply:
done